File: Exercicios-Beecrowd/C#/1006.cs
using System;

class Mult{
    static void Main(string[] args){
        double a = Convert.ToDouble(Console.ReadLine());
        double b = Convert.ToDouble(Console.ReadLine());
        double c = Convert.ToDouble(Console.ReadLine());
        a *= 2;
        b *=3;
        c *=5;
        double soma = (a+b+c)/10;
        string media = soma.ToString("F1");
        
        Console.WriteLine("MEDIA = "+ media);

    }
}

File: Exercicios-Beecrowd/Kotlin/1006.kt
import java.util.*

fun main(args: Array<String>) {

    val valor1= readLine()
    val valor2= readLine()
    val valor3= readLine()
    if (valor1 != null && valor2!=null && valor3!= null){
        val nota1=valor1.toDouble()
        val nota2=valor2.toDouble()
        val nota3=valor3.toDouble()
        val resultado= ((nota1*2)+(nota2*3)+(nota3*5))/10
        val formatado = ("%.1f".format(resultado))
        println("MEDIA = $formatado")
    }
	
}